### DISCUSSION TITLE
Qual ferramenta vocês usam para capturar erros de runtime do JavaScript?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#492] [Perguntas e Respostas]
Quando o site está em produção, qual ferramenta vocês usam para saber que algum cliente está tendo erros no browser dele seja la qual for o browser, eu já usei o [sentry](https://sentry.io/for/javascript/) porém gostaria de ouvir de vocês sobre alguns outros.

> edit coloquei uma descrição

**Comment by RatoX** [comment]
Em produção? [Rollbar](https://rollbar.com/)

**Comment by munizart** [comment]
Quando a pagina não está aquele turbilhão de scripts gosto de usar o próprio console, já que em quase todos podemos settar uma opção parecida com ```Pause on Exceptions```

**Comment by evansantos** [comment]
Eu estou muito interessado em usar o [opbeat](http://opbeat.com), não sei se alguém aqui já usou

**Comment by eduardojmatos** [comment]
@felquis usei sempre o airbrake pra track de erros. Funciona que é uma beleza, mas é pago.

**Comment by marceloogeda** [comment]
@felquis já utilizei o [Tracekit](https://github.com/csnover/TraceKit) e o [Sentry](https://sentry.io/for/javascript/), ambos atenderam perfeitamente as necessidades. O Sentry é pago, mas é possível roda-lo local, só precisa analisar se é viável a mantenabilidade (custo de máquinas e etc).
